### PR TITLE
Facund/touch swipe

### PR DESCRIPTION
**Pull request by facundfortuny**
Add swipe on touch helper and Update Events

### CODE REVIEW

File: src/touch.js [hunk 1]
```diff
@@ -24,62 +23,64 @@ define([
...
+        return (this.touchStartX === posX) && (this.touchStartY === posY);
     };
-
     /**
-     * A helper function to prevent any defaults on a given event
+     * Check if touch has moved position to determine swipe.
      * @memberOf module:touch
      *
      * @protected
-     * @param {Object} event An event object passed from the browser such as mouseover or click
```
Comment (jonman) [inline src/touch.js]: Can you check that your jsdocs are up to date?

File: src/touch.js [hunk 2]
```diff
@@ -103,29 +104,103 @@ define([
      * ```
      */
     Touch.prototype.tap = function (ele, callback) {
-        this.events.addListeners(ele, 'touchstart', this._touchStartHandler(callback));
-        this.events.addListeners(ele, 'touchend click', this._touchEndHandler(callback));
+        var $this = this;
+
+        this.events.addListeners(ele, 'click', $this._touchEndHandler(callback));
```
Comment (jonman) [inline src/touch.js]: Is it possible to store the various handlers in a `var`?
```js
var $this = this,
  startHandler = $this._touchStartHandler(),
  endHandler = $this.touchEndHandler(callback);

..

this.events.addListeners(ele, 'click', endHandler);

```

**Comment by nwhite89** [comment]
Could you ensure to run ``npm run docs`` to ensure that the docs within the ``README.md`` are updated please :+1:

**Comment by nwhite89** [comment]
Looks good for merge @jonman?

**Comment by jonman** [comment]
:+1:

**Pull request merged**